EveripediaNetwork/ep-ui
# Fixed toc not displaying for h2s

# Fixed TOC Not Displaying for H2s

_This PR fixes TOC not displaying for h2s_

## How should this be tested?

1. Visit https://iq-wiki-git-fix-toc-prediqt.vercel.app/wiki/binance-peg-busd/ and confirm the TOC displays
2. Compare with https://iq.wiki/wiki/binance-peg-busd/

## Notes or observations

_Both links contain same wiki, the first is just a copy of the second_

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3112
